src/components/application_manager/test/application_manager_impl_test.cc

```diff
@@ -85,7 +86,7 @@ sync_primitives::ConditionalVariable state_condition_;
 
 class ApplicationManagerImplTest : public ::testing::Test {
  protected:
-  void SetUp() OVERRIDE {
+  virtual void SetUp() OVERRIDE {
```

Comment:
@AGaliuzov, thanks for found it. I've already delete and it's appearing here in next commit.